test/subscription_manager.ts

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
`identityToKey` & `genSubscriptionKey` are private
`subscriptionExists` is used in every test, so it's tested implicitly (example: `registers a subscription` test)